screens/Playlist.js

```diff
@@ -0,0 +1,282 @@
+    }
+  }
+  if (isLoading) {
+    return (
+      <View style={[styles.container, { flex: 1, paddingTop: insets.top }]}>
+        <Text bold style={styles.headerTitle}>
+          MEDIA
+        </Text>
+        <Spinner />
+      </View>
+    );
+  }
+
+  if (isError) {
+    return (
+      <View style={[styles.container, { flex: 1, paddingTop: insets.top }]}>
+        <Text bold style={styles.headerTitle}>
+          MEDIA
+        </Text>
+        <View
+          style={[
+            styles.container,
+            { alignItems: 'center', justifyContent: 'center', padding: 16 },
+          ]}
+        >
+          <Title style={[{ textAlign: 'center' }]}>
+            Oh no! There was an error connecting to YouTube 😞
+          </Title>
+          <Text style={[{ textAlign: 'center', color: Colors.darkGray }]}>
+            {errorMessage}
+          </Text>
+          <Button
+            title={'Retry'}
+            style={styles.notesButton}
+            onPress={() => {
+              setError(false);
+              setLoading(true);
+              getVideos();
+            }}
+          />
+        </View>
+      </View>
+    );
+  }
+
+  return (
+    <HeaderHeightContext.Consumer>
+      {(headerHeight) => (
+        <View style={[styles.mainContainer, { paddingTop: headerHeight }]}>
+          <ScrollView>
+            <Image
+              source={{ uri: playlistURI }}
+              style={styles.image}
+              resizeMode="cover"
+            />
+            <Text style={styles.description} numberOfLines={4}>
+              {data.length ? data[0].description : ''}
+            </Text>
+            <Subtitle style={styles.subtitle}>IN THIS SERIES</Subtitle>
+            {/* {
+            data.map(({
+              item: {
+                id,
+                publishDate,
+                title,
+                description,
+                thumbnails: { maxres = {} } = {},
+              } = {},
+            }) => (
+              <TouchableHighlight
+                underlayColor="transparent"
+                onPress={() => {
+                  Amplitude.logEvent(`OPEN ${title}`);
+                  Linking.openURL(`https://www.youtube.com/watch?v=${id}`);
+                }}
+              >
+                <View style={styles.item}>
+                  <Image
+                    source={{ uri: maxres.url }}
+                    style={styles.thumbnailStyle}
+                    resizeMode="cover"
+                  />
+                  <View style={{flex:1, flexDirection: "column"}}>
+                    <Heading style={styles.title}>
+                      {title}
+                    </Heading>
+                    <Text style={[styles.description, {paddingTop: 8}]}>
+                    {new Date(publishDate).toLocaleDateString()}
+                    </Text>
+                    <View style={{flex:1}}/>
+                  </View>
+                </View>
+              </TouchableHighlight>
+            ))
```

Comment:
@dereckquock I don't know why this map replacement for the flatlist isn't working.